### PR TITLE
CIV-12895 CCJ url for settlement

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-12895

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

File: src/main/routes/features/dashboard/claimantDashboardController.ts [hunk 1]
```diff
@@ -22,9 +26,9 @@ claimantDashboardController.get(DASHBOARD_CLAIMANT_URL, (async (req: AppRequest,
       claim = await getClaimById(userId, req);
       dashboardId = userId;
     } else {
-      claim = await getClaimById(claimId, req, true);
+      claim = await civilServiceClient.retrieveClaimDetails(claimId, req);
       caseRole = claim.isClaimant()?ClaimantOrDefendant.CLAIMANT:ClaimantOrDefendant.DEFENDANT;
```
Comment (laarmada) [inline src/main/routes/features/dashboard/claimantDashboardController.ts:30]: If im not wrong here this vble will be undefined: "caseRole". Please test the issue as user.

**Review by laarmada — CHANGES_REQUESTED** [review]
Please show the screen shot of the claim with the new changes "/:id/claimant" I think you are missing some data here...

File: src/main/routes/features/dashboard/claimantDashboardController.ts (hunk 1 above)
Comment (jeswanth-hmcts) [reply src/main/routes/features/dashboard/claimantDashboardController.ts:30]: In the retriveClaimDetails, we are calling an API call for roles setting  the role  in  case_data , in cui translation , we setting the value  in claim , and tested the scenario , it is working fine. PF the SS for reference
image

Comment (ElifDenizEsea) [reply src/main/routes/features/dashboard/claimantDashboardController.ts]: It was coming from master I rolled back it now.

**Comment by ElifDenizEsea** [comment]
dashboard

**Review by laarmada — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**